src/ixdat/measurements.py

```diff
@@ -0,0 +1,375 @@
+            return plotter.plot_measurement(self, *args, **kwargs)
+        return self.plotter.plot(*args, **kwargs)
+
+    def export(self, exporter=None, *args, **kwargs):
+        """Export the measurement using its exporter (see its Exporter for details)"""
+        if exporter:
+            return exporter.export_measurement(self, *args, **kwargs)
+        return self.exporter.export(*args, **kwargs)
+
+    def __add__(self, other):
+        """Addition of measurements appends the series and component measurements lists.
+
+        Adding results in a new Measurement. If the combination of the two measurements'
+        techniques is a recognized hyphenated technique, it returns an object of that
+        technique's measurement class. Otherwise it returns an object of Measurement.
+        metadata, sample, and logentry come from the first measurement.
+
+        An important point about addition is that it is almost but not quite associative
+        and transitiry i.e.
+        A + (B + C) == (A + B) + C == C + B + A   is not quite true
+        Each one results in the same series and component measurements. They will even
+        appear in the same order in A + (B + C) and (A + B) + C. However, the technique
+        might be different, as a new technique might be determined each time.
+
+        Not also that there is no difference between hyphenating (simultaneous EC and
+        MS datasets, for example) and appending (sequential EC datasets). Either way,
+        all the raw series (or their placeholders) are just stored in the lists.
+        TODO: Make sure with tests this is okay, differentiate using | operator if not.
+        """
+        obj_as_dict = self.as_dict()
+        new_name = self.name + " AND " + other.name
+        new_technique = self.technique + " AND " + other.technique
+
+        from .techniques import TECHNIQUE_CLASSES
+
+        if new_technique in TECHNIQUE_CLASSES:
+            cls = TECHNIQUE_CLASSES[new_technique]
+        elif self.__class__ is other.__class__:
+            cls = self.__class__
+        else:
+            cls = Measurement
+
+        new_series_list = self.series_list
+        new_series_list.append(other.series_list)
+        new_component_measurements = self.component_measurements or [self]
+        new_component_measurements.append(other.component_measurements or [other])
+        obj_as_dict.update(
+            name=new_name,
+            series_list=new_series_list,
+            component_measurements=new_component_measurements,
+        )
+        return cls.from_dict(obj_as_dict)
+
+
+#  ------- Now come a few module-level functions for series manipulation ---------
+#  TODO: move to a .build module or similar
+
+
+def append_series(series_list):
+    """Return series appending series_list relative to series_list[0].tseries.tstamp"""
+    s0 = series_list[0]
+    if isinstance(s0, TimeSeries):
+        return append_tseries(series_list)
+    elif isinstance(s0, TimeSeries):
```

Comment:
If and elif seems to be the same

Reply:
True. Second should of course have been:
```
    elif isinstance(s0, ValueSeries):
        return append_vseries_by_time(series_list)
```
Now fixed.